edx_solutions_api_integration/courses/views.py

```diff
@@ -1044,6 +1044,8 @@ def post(self, request, course_id):
             try:
                 email = request.data['email']
                 existing_user = User.objects.get(email=email)
+                CourseEnrollment.enroll(existing_user, course_key)
```

Comment:
Can we further optimize the code, to create course enrollment which caters to both email and user id cases?